src/css/states/play.less

```diff
@@ -0,0 +1,5 @@
+.jwplayer.jw-state-play {
+    .jw-controlbar .jw-controlbar--left-group .jw-play-state-icon {
+        .jwplayer .jw-icon-pause;
```

Comment:
This mixin adds the styles of .jwplayer .jw-icon-pause class to the play/pause toggle icon in the control bar.  In this case it's just setting the content of the after for the icon code of the last element to be the pause icon.  Doing this gets us a semantic way to say which icon to use for that element's :after value without resorting to setting the character code statically.